collections/hub/provisioning/new_participants/new_dfsp.json

```diff
@@ -91,6 +91,51 @@
               }
             ]
           }
+        },
+        {
+          "id": 3,
```

Comment:
I think there is a duplicate testcase ID `3`.

Reply:
this is request id, not testcase id

Reply:
Oh ok, you added as a request. Got it